src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionVisibilityOptions.tag

```diff
@@ -40,7 +41,8 @@
         </ul>
     </div>
 </div>
-<div class="col-sm-12 margin-bottom-15px background-color-light-green">
+<!-- Fix for collapsing margin problem. Reference: http://stackoverflow.com/questions/6204670 -->
```

Comment:
If you could identify the common issues you faced while you changed these pages, it would be very helpful